frontend/src/config/Icons.ts

```diff
@@ -32,6 +32,18 @@ export type IconType = {
 // or from the font awesome site: https://fontawesome.com/icons
 const mutIcons = {
   istio: {
+    arrowLeftRight: {
+      ascii: '\u21c4 ',
+      name: 'arrowLeftRight',
+      text: 'Arrow Left Right',
+      type: 'fa'
+    },
+    arrowLeftRightV: {
+      ascii: '\u21c5 ',
+      name: 'arrowLeftRightV',
+      text: 'Arrow Left Right Vertical',
+      type: 'fa'
+    },
```

Comment:
Since these aren't really icons and don't actually represent an Istio feature/component, maybe we can add a new section below, under this `istio` set of icons, that's dedicated to just unicode chars we want to use, that don't have an associated icon.  Also, let's use the exact naming as used on the site:

```
  },
  unicode: {
    // these are not fully defined icons, just unicode characters.
    // see https://symbl.cc/en/unicode-table/
    arrowRightOverLeft: {
      ascii: '\u21c4 ',
      name: 'RightwardsArrowOverLeftwardsArrow',
      text: 'Rightwards Arrow Over Leftwards Arrow'
    },
    arrowUpLeftofDown: {
      ascii: '\u21c5 ',
      name: 'UpwardsArrowLeftwardsofDownwardsArrow',
      text: 'Upwards Arrow Leftwards of Downwards Arrow'
    }
  }
```

Reply:
I've done some tests and maybe is not really nice on this side: 

![image](https://github.com/user-attachments/assets/63bd82b5-5f30-4b5e-9599-82fd552298cf)

Maybe a bit closer?

![image](https://github.com/user-attachments/assets/fb0e8de5-d4a1-42f9-aaa6-0d68ed0cb533)

I also added the arrows in the up/down direction so the icon is switched when you click on it, it makes more apparent to the user that things are changed:


[Screencast from 24-09-24 16:56:37.webm](https://github.com/user-attachments/assets/ce2fa486-4da5-4d93-ba5c-a27313935915)

Reply:
I actually like it on the left :-)  Let's show it to Foday tomorrow and see if he has an opinion.  Either way, I like it better than tabs!